kanidm_proto/src/v1.rs

```diff
@@ -310,7 +310,7 @@ pub enum CredentialDetailType {
     GeneratedPassword,
     Webauthn(Vec<String>),
     /// totp, webauthn
-    PasswordMfa(bool, Vec<String>),
+    PasswordMfa(bool, Vec<String>, bool),
```

Comment:
You may want this to be a number of how many codes are remaining so they know they may need to reset it soon if they are low.